ChopChop/Storage/StorageManager.swift

```diff
@@ -228,6 +229,61 @@ extension StorageManager {
     }
 }
 
+// MARK - Firebase operations
+extension StorageManager {
+    
+    func publishRecipe(recipe: inout Recipe, userId: String) throws {
+        var cuisine = ""
+        if let categoryId = recipe.recipeCategoryId {
+            cuisine = (try? (fetchRecipeCategory(id: categoryId)?.name) ?? "") ?? ""
```

Comment:
i think the RecipeCategory may cause quite alot of issues, needa discuss tog

Reply:
<img width="938" alt="Screenshot 2021-04-02 at 12 59 06 AM" src="https://user-images.githubusercontent.com/54176678/113328258-a2068300-934e-11eb-8e6f-f59d186b7969.png">
I think update can consider just 'saveRecipe' instead of 'publishRecipe' and 'updateOnlineRecipe'

Firebase does that check for u

Reply:
Not sure what you mean here but if I use setData but adding new recipe then i would need to supply a documentID instead of have firebase generate one for me